src/modules/site/templates/data/vbrowser.html

```diff
@@ -294,9 +294,10 @@ <h3 class="text-center">Variant Impact</h3>
          url: "{{ url_for('api_gene.api_search_combined', query='') }}" + gene,
          method: "GET",
          contentType: 'application/xml',
-         }).done(function(msg) {
+         }).done(function(results) {
           row = Array();

```

Comment:
is `row` being used globally elsewhere? If so, we should refactor, otherwise, consider removing it here.